paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/GenericTextFieldConfig.kt

```diff
@@ -0,0 +1,30 @@
+package com.stripe.android.paymentsheet.elements
+
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import com.stripe.android.paymentsheet.ElementType
+
+internal class GenericTextFieldConfig(
```

Comment:
The naming of the config seems a bit confusing, why is a GenericXXX extending a XXX? should we change the interface name somehow?

Reply:
I could name it a PlainTextFieldConfig or SimpleTextFieldConfig or SimpleTextFieldConfigImpl?

Would any of those improve clarity?

Reply:
Thanks, `SimpleTextFieldConfig` sgtm!